hoot-services/pom.xml

```diff
@@ -136,7 +136,7 @@
                     parallelism, but haven't been able to get them to work...so forking instead. -->
                     <!-- parallel>methods</parallel>
                     <useUnlimitedThreads>true</useUnlimitedThreads -->
-                    <forkCount>4</forkCount>
+                    <forkCount>16</forkCount>
```

Comment:
@bwitham @brianhatchl 

Created a new issue to allow forkCount of maven-sure-fire plugin to be set to $(nproc)
https://github.com/ngageoint/hootenanny/issues/814

Dmitriy